internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -259,6 +259,10 @@ func (c *CodePipeline) GetPipelineState(name string) (*PipelineState, error) {
 	}, nil
 }
 
+func (sa StageAction) humanStringAction() string {
+	return "    " + sa.Name + "\t" + sa.Status + "\n"
```

Comment:
A tiny recommendation: We can remove the newline character at the end of this string to describe a single StageAction, instead when we join the `formattedActions` we can join on `"\n"`.

This way can provide more flexibility on how people want to glue together these human strings.

Reply:
Ah, I like that!

Reply:
Haha--I put some highly unlikely/impossible(?) test cases in my tests and one of them found that if there is a Stage with NO Actions, there will be an extra line between that Stage and the next, because of the moved `\n`. I think this is okay, because it probably won't happen, but also because it actually draws attention to the fact that there's a stage with no Action.